EnglishTranslation-master/ETC.tsv

```diff
@@ -27608,14 +27608,14 @@ ETC_20170330_027602	Your companion is not in active mode.	컴패니언이 휴면
 ETC_20170330_027603	Cannot log out during the matching process.	매칭 요청 중에는 로그아웃 할 수 없습니다.				
 ETC_20170330_027604	Attribute point limit exceeded.	특성 포인트 보유 한도를 초과합니다.				
 ETC_20170330_027605	Not enough attribute points.	특성 포인트가 부족합니다.				
-ETC_20170330_027606	Cannot move to another map during the matching process.	매칭 요청 중에는 맵 이동을 할 수 없습니다.				
+ETC_20170330_027606	You cannot move to another map during a match request.	매칭 요청 중에는 맵 이동을 할 수 없습니다.				
```

Comment:
This is not a match request, it's the matching process for TBL; the original translation was correct.